eZ/Bundle/EzPublishCoreBundle/Imagine/BinaryLoader.php

```diff
@@ -56,6 +57,17 @@ public function find($path)
             );
         } catch (NotFoundException $e) {
             throw new NotLoadableException("Source image not found in $path", 0, $e);
+        } catch (InvalidBinaryFileIdException $e) {
+            $message =
+                "Source image not found in $path. Repository images path are expected to be " .
+                "relative to the var/<site>/storage/images directory";
+
+            $suggestedPath = preg_replace('#var/[^/]+/storage/images/#', '', $path);
+            if ($suggestedPath) {
+                $message .= "\nSuggested value: '$suggestedPath'";
+            }
+
+            throw new NotLoadableException($message);
```

Comment:
I'm not sure what this means exactly, but I have swapped out the catch statements, and made the final one general, so that we always throw a `NotLoadableException` (it is the one other loaders use, even though it is not part of the interface doc.

Does it work for you ?